src/core/metallicroughnessmaterial.cpp

```diff
@@ -322,6 +323,15 @@ WrappedSignal<OutputType> wrapParameterSignal(MetallicRoughnessMaterial *self, S
     \since 1.1
  */
 
+/*!
+    \property morphController
+
+    The morph controller instance used to control morph targets weights.
+    This can be null if a material is not used without morph targets.
```

Comment:
"a material is not used without morph targets" means a material that uses morph targets?

Reply:
Fixed, should be "with" and not "without".